devtool.md

```diff
@@ -0,0 +1,167 @@
+---
```

Comment:
This devtool.md file can be deleted. It does only apply to our former build system and not the OpenHarmony based one.